AlphaWallet.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,16 +1,14 @@
 {
-  "object": {
-    "pins": [
-      {
-        "package": "WalletConnect",
-        "repositoryURL": "https://github.com/WalletConnect/WalletConnectSwiftV2",
-        "state": {
-          "branch": null,
-          "revision": "5ea0265d34f18ec48d71fc299adacecdacc904bb",
-          "version": "0.5.0"
-        }
+  "pins" : [
```

Comment:
Can you rebase master (or one of the earlier ones which still builds like `31d3fe25affce53c9ce42464b653b9bd97b87537`)? It'll get rid of this in the PR.